client/layout/global-sidebar/footer.tsx

```diff
@@ -60,3 +61,23 @@ export const GlobalSidebarFooter: FC< {
 		</SidebarFooter>
 	);
 };
+
+export const GlobalSiteSidebarFooter: FC< {
+	selectedSiteSlug: string;
+	translate: LocalizeProps[ 'translate' ];
+} > = ( { selectedSiteSlug, translate } ) => {
+	return (
+		<SidebarFooter>
+			<a
+				href={ `https://${ selectedSiteSlug }/wp-admin/plugins.php` }
+				className="sidebar__footer-wpadmin tooltip tooltip-top"
+				title={ translate( 'WP Admin' ) }
+				data-tooltip={ translate( 'WP Admin' ) }
+			>
+				<Icon icon={ wordpress } className="wpicon" />
+				{ translate( 'WP Admin' ) }
+				<Icon icon={ external } className="external" />
```

Comment:
With this icon, I think we have to open the wp-admin on the new tab.

Reply:
I initially felt the same, but I believe it's preferable to remain in the same tab for this scenario. 
W3C generally advises against opening new tabs, and it specifies that it should be avoided unless in explicit situations. I thought our scenario did not meet the criteria for opening a new tab as outlined by the WCAG here: https://www.w3.org/TR/WCAG20-TECHS/G200.html.

>- Opening a page containing context-sensitive information, such as help instructions, or an alternate means of completing a form, such as a calendar-based date picker, will significantly disrupt a multi-step workflow, such as filling in and submitting a form, if the page is opened in the same window or tab.
>- The user is logged into a secured area of a site, and following a link to a page outside of the secured area would terminate the user's logon. In this case opening external links in an external window allows the user to access such references while keeping their login active in the original window.

While it's debatable, I've concluded that opening a new tab, even when an external icon is present, is not the best approach in this instance.

- [Should Links Open In New Windows? — Smashing Magazine](https://www.smashingmagazine.com/2008/07/should-links-open-in-new-windows/)
- [External Links, New Tabs, and Accessibility / Coder's Block](https://codersblock.com/blog/external-links-new-tabs-and-accessibility/)
- [When to use target="_blank" | CSS-Tricks - CSS-Tricks](https://css-tricks.com/use-target_blank/)

Reply:
Sounds good! I agree to remove the external icon for now 👍